content/news/XXXX-XX-XX-mixxx-2-3-0-released.md

```diff
@@ -0,0 +1,138 @@
+title: Mixxx 2.3.0 released
+authors: Jan Holthuis
+tags: 2.3, 2.3.0, release announcement
+status: draft
+comments: yes
+
+We are proud to announce the release of Mixxx 2.3.0!
+If you want to give it a spin, you can now [download it]({filename}/pages/download.md#stable) on Windows, macOS and Linux.
+
+This release brings a lot of new features and bugfixes:
+
+Prepare your DJ set using [hotcue colors & labels]({filename}/news/2020-08-25-new-in-2-3-hotcue-colors.md), mark [intro/outro sections]({filename}/news/2020-07-09-intro-outro-sections.md) in your tracks and add [track colors]({filename}/news/2020-10-24-new-in-2-3-track-colors.md) for easier library navigation.
+Our new multithreaded analysis and [more accurate key detection]({filename}/news/2021-04-08-new-in-2-3-keyfinder.md) will improve the preparation process further.
+
+Mixxx 2.3.0 comes with a new default skin:
+"LateNight" underwent a massive redesign and replaces "Deere" as default skin, so check out the [screenshots page]({filename}/pages/screenshots.md).
+
+If you're a Rekordbox or Serato user, switching to Mixxx becomes a lot easier:
+You can now play tracks directly from USB drives that contain [Rekordbox]({filename}/news/2020-07-20-new-in-2-3-rekordbox-support.md) and [Serato](2021-02-08-new-in-2-3-serato-support.md) libraries.
+
+We also added support for recording/streaming in the [Opus](https://opus-codec.org/) and [HE-AAC](https://www.iis.fraunhofer.de/en/ff/amm/broadcast-streaming/heaac.html) codecs, introduced [deck cloning]({filename}/news/2020-11-15-new-in-2-3-deck-clone.md), polished the library and preferences (including the controller workflow) and [switched to the CMake build system generator]({filename}/news/2020-12-14-scons-cmake-migration.md).
```

Comment:
Can we move the CMake part to the infrastucture part below. Than we don't mix dev features with user features.

Reply:
I agree. Apart from that this is a good read!